Django_data/templates/Game/Stats.html

```diff
@@ -19,68 +19,74 @@
             <h1 class="gameStats">Player Stats</h1>
           </div>
           
-                  <div class="section-stats">
-                    <span class="stats-title">Winrate: {{ win_rate }}</span>
-                    <img src="{%static 'images/duck-ducky.gif'%}"></img>
+                  <div class="section-stats-center">
+                    {% if win_rate >= 0 and win_rate <= 33 %}
+                      <span class="WR-title low-ranked">Winrate: {{ win_rate }}</span>
+                      <img class="ranked-gif" src="{%static 'images/sad.gif'%}"></img>
+                    {% elif win_rate > 33 and win_rate <= 66 %}
+                      <span class="WR-title medium-ranked">Winrate: {{ win_rate }}</span>
+                      <img class="ranked-gif" src="{%static 'images/duck-ducky.gif'%}"></img>
+                    {% elif win_rate > 66 and win_rate <= 100 %}
```

Comment:
{% elif win_rate > 0.66 and win_rate <= 1 %}

Reply:
Merciiii 👍